pkg/apis/extensions/v1beta1/types.go

```diff
@@ -967,10 +967,11 @@ type PodSecurityPolicySpec struct {
 	// will not be forced to.
 	ReadOnlyRootFilesystem bool `json:"readOnlyRootFilesystem,omitempty" protobuf:"varint,14,opt,name=readOnlyRootFilesystem"`
 	// SeccompProfiles lists the allowed profiles that may be set for the pod or
-	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles may
-	// be specifid by the pod or container.	The wildcard '*' may be used to allow all profiles.  When
-	// used to generate a value for a pod the first non-wildcard profile will be used as
-	// the default.
+	// container's seccomp annotations.  An unset (nil) or empty value means that no profiles
+	// are allowed to be set and that a default (docker/default) will be set by the strategies.
+	// A non-empty value contains the list of allowed profiles and the first non-wildcard (*) should
```

Comment:
I guess I'm confused on what you're going for here.  I think there are a couple of options based on the last round of feedback:

1.  PSP provides the default if PSP.seccomp setting is nil - right now this is `docker/default`.
2.  PSP doesn't provide a default if PSP.seccomp is nil - the kubelet's container runtime code provides it

2 may be better than what I have here since it is runtime aware.  It could be a follow up.  Then we are back in the original state of this PR before these feedback changes where PSP tolerates nil and only supplies the default if the PSP.seccomp has values  :smile: